Week1/homework/create-tables.js

```diff
@@ -0,0 +1,29 @@
+const mySQL = require('mysql');
+const connection = mySQL.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'myWorld',
+});
+
+connection.connect();
+
+const create_country_query =
+  "create table country (Name VARCHAR(20), Continent enum('Asia', 'Europe', 'North America', 'Africa', 'Oceania', 'Antarctica', 'South America'), Region text, Surface_Area float, IndepYear date, Population INT, Life_Expectancy INT(3), GNP float, GNPOld float, Local_Name text, Government VARCHAR(25), Head_Of_State VARCHAR(60), Capital text)";
+
+connection.query(create_country_query, function(error, results, fields) {
+  if (error) {
+    throw error;
+  }
+  console.log('your table is' + results);
+});
+
+const create_city_query =
+  'create table city (name varchar(20), country_code char(3), district text, population int)';
+connection.query(create_city_query, function(error, results, fields) {

```

Comment:
Don't forget the ID :) But Its good otherwise

Reply:
ok i added the id as well, but is there another way to automatize the id without typing it all the time?

Reply:
Yes you can use auto increment so it will be incremented by default. But you need to add which column name you fill in your insert to use it :)